ocp_resources/node_health_check.py

```diff
@@ -8,3 +8,66 @@ class NodeHealthCheck(Resource):
     """
 
     api_group = Resource.ApiGroup.REMEDIATION_MEDIK8S_IO
+
+    def __init__(
+        self,
+        node_role="worker",
```

Comment:
do not set a default here

Reply:
after installing operator if i check yaml , default yaml has:

          "selector": {
              "matchExpressions": [
                {
                  "key": "node-role.kubernetes.io/worker",
                  "operator": "Exists"
                }
              ]
            },

Reply:
you should not hard code the resource defaults; what if this is changed?

Reply:
it is mandatory to have this field.

Reply:
does that mean node_role="worker" should have node_role=None?

Reply:
@rnetser Hi, ^

Reply:
yes